ios/browser/api/ads/brave_ads.mm

```diff
@@ -1145,6 +1145,17 @@ - (void)loadAdsResource:(const std::string&)id
   callback(/* success */ false, "");
 }
 
+- (void)clearScheduledCaptcha {
+}
+
+- (void)getScheduledCaptcha:(const std::string&)payment_id
+                   callback:(ads::GetScheduledCaptchaCallback)callback {
+}
```

Comment:
Does this get called on iOS ever and wait for a response from the callback? If it does you may have to call the callback even with empty info

Reply:
No, iOS is excluded via a buildflag and never calls this function, but I added a call to the callback with empty info just to play it safe - thanks!